sites/skiplinks/includes/skiplinks.html

```diff
@@ -1,9 +1,9 @@
-<nav><ul id="wb-tphp">
-	<li class="wb-slc"><a class="wb-sl" href="#wb-cont">{{ i18nText-skipCont }}</a></li>
-{%- unless page.noFooterAbout and page.noFooterSite -%}
-	<li class="wb-slc visible-sm visible-md visible-lg"><a class="wb-sl" href="#wb-info">{{ i18nText-skipAbout }}</a></li>
-{%- endunless -%}
-{%- if page.secondarymenu -%}
-	<li class="wb-slc visible-md visible-lg"><a class="wb-sl" href="#wb-sec">{{ i18nText-skipSection }}</a></li>
-{%- endif -%}
-</ul></nav>
+<nav>
+	<ul id="wb-tphp">
+		<li class="wb-slc"><a class="wb-sl" href="#wb-cont">{{ i18nText-skipCont }}</a></li>
+		<li class="wb-slc visible-sm visible-md visible-lg"><a class="wb-sl" href="#wb-info">{{ i18nText-skipAbout }}</a></li>
+		{%- if page.secondarymenu -%}
+		<li class="wb-slc visible-md visible-lg"><a class="wb-sl" href="#wb-sec">{{ i18nText-skipSection }}</a></li>
+		{%- endif -%}
+	</ul>
+</nav>
```

Comment:
We should note this change into our Footer documentation page and in the skip link documentation page